src/common/schemas/input/compute-ledger-hash.json

```diff
@@ -5,7 +5,8 @@
   "properties": {
     "ledger": {
       "$ref": "getLedger",
-      "description": "The ledger header to hash."
+      "description": "The ledger header to hash.",
+      "position": 0
```

Comment:
I've made that way because we can't use jayson method - babel doesn't put
parameters that have default values into function definition - it checks
`arguments` value to get that parameters.
7 груд. 2015 21:21 "Chris Clark" <notifications@github.com> пише:

> In src/common/schemas/input/compute-ledger-hash.json
> <https://github.com/ripple/ripple-lib/pull/663#discussion_r46863460>:
>
> > @@ -5,7 +5,8 @@
> >    "properties": {
> >      "ledger": {
> >        "$ref": "getLedger",
> > -      "description": "The ledger header to hash."
> > +      "description": "The ledger header to hash.",
> > +      "position": 0
>
> This is not a very robust solution because it can get out of sync with the
> actual method signatures. I think it would be better to use the same
> technique that jayson uses to get the method parameter ordering.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ripple/ripple-lib/pull/663/files#r46863460>.
>